packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.controller.js

```diff
@@ -28,13 +29,17 @@ export default class SupportNewIssuesFormController {
   }
 
   $onInit() {
+    this.defaultServiceName = this.serviceName;
```

Comment:
what is the use of defaultServiceName? can't we just use a single serviceName var? if not please add a little comment for clarity because i find it a little bit confusing that we have two vars that describe the same value

Reply:
`this.serviceName` is the name of the service to get tickets fort
`this.defaultServiceName` the name that was given to the component (by the URL for example). This way, I know if the current name of the service has changed compared to what the user gave to the component

The component acts differently if you give it a  `serviceName` and `serviceTypeName`: if you select a `category` in the `category` select box, then the `serviceName` and `serviceTypeName` won't be reset ! If no `serviceName` and `serviceTypeName` are given, then they are both reset